### PR TITLE
First Japanese l10n work for release-1.13

### PR DESCRIPTION
**Pull request by cstoku**
First Japanese l10n work for release-1.13.(include minimum l10n contents)

<details>
  <summary><b>Change List</b></summary>
  <ul>
    <li>Translate content/ja/docs/home/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/11569" target="_blank">#11569</a>)</li>
    <li>[ja] add basic files for 1.13 (<a href="https://github.com/kubernetes/website/pull/11571" target="_blank">#11571</a>)</li>
    <li>Translate content/ja/docs/setup/custom-cloud/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/11572" target="_blank">#11572</a>)</li>
    <li>ja-trans: Translate setup/independent/_index.md (<a href="https://github.com/kubernetes/website/pull/11573" target="_blank">#11573</a>)</li>
    <li>Translate content/en/docs/setup/on-premises-vm/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/11574" target="_blank">#11574</a>)</li>
    <li>Translate content/ja/docs/setup/release/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/11576" target="_blank">#11576</a>)</li>
    <li>Dev 1.13 ja.1 tutorials kubernetes basics scale (<a href="https://github.com/kubernetes/website/pull/11577" target="_blank">#11577</a>)</li>
    <li>translate deprecated state description (<a href="https://github.com/kubernetes/website/pull/11578" target="_blank">#11578</a>)</li>
    <li>Translate content/ja/docs/tutorials/kubernetes-basics/update/_index.m… (<a href="https://github.com/kubernetes/website/pull/11579" target="_blank">#11579</a>)</li>
    <li>ja-trans: Translate content/ja/docs/tutorials/kubernetes-basics/explore/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/11580" target="_blank">#11580</a>)</li>
    <li>[ja] add ja section (<a href="https://github.com/kubernetes/website/pull/11581" target="_blank">#11581</a>)</li>
    <li>Translate content/ja/docs/setup/turnkey/_index.md (<a href="https://github.com/kubernetes/website/pull/11582" target="_blank">#11582</a>)</li>
    <li>Translated Tutorials/Learn Kubenetes Basics/Deploy an App in Japanese. (<a href="https://github.com/kubernetes/website/pull/11583" target="_blank">#11583</a>)</li>
    <li>trans-jp: tutorials/kubernetes-basics/expose/_index.md (<a href="https://github.com/kubernetes/website/pull/11584" target="_blank">#11584</a>)</li>
    <li>ja-trans: translate content/ja/_index.html (<a href="https://github.com/kubernetes/website/pull/11585" target="_blank">#11585</a>)</li>
    <li>[ja] add ja.toml (<a href="https://github.com/kubernetes/website/pull/11595" target="_blank">#11595</a>)</li>
    <li> Fix the build doesn't pass at dev-1.13-ja.1 (<a href="https://github.com/kubernetes/website/pull/11609" target="_blank">#11609</a>)</li>
    <li>Translate stable state description (<a href="https://github.com/kubernetes/website/pull/11642" target="_blank">#11642</a>)</li>
    <li>ja-trans: tutorials/hello-minikube.md (<a href="https://github.com/kubernetes/website/pull/11648" target="_blank">#11648</a>)</li>
    <li>ja-trans: add docs/_index.md (cherry-pick from dev-1.12-ja.1) (<a href="https://github.com/kubernetes/website/pull/11721" target="_blank">#11721</a>)</li>
    <li>Remove copied docs/index.md by mistake. (<a href="https://github.com/kubernetes/website/pull/11735" target="_blank">#11735</a>)</li>
    <li>Translate alpha state description (<a href="https://github.com/kubernetes/website/pull/11753" target="_blank">#11753</a>)</li>
    <li>Translate beta state description (<a href="https://github.com/kubernetes/website/pull/12023" target="_blank">#12023</a>)</li>
    <li>ja-trans: Translate content/ja/docs/tutorials/kubernetes-basics/create-cluster/_index.md in Japanese (<a href="https://github.com/kubernetes/website/pull/12059" target="_blank">#12059</a>)</li>
    <li>[ja] translate case-studies (<a href="https://github.com/kubernetes/website/pull/12060" target="_blank">#12060</a>)</li>
    <li>[ja] fix home content (<a href="https://github.com/kubernetes/website/pull/12062" target="_blank">#12062</a>)</li>
    <li>[ja] translate what-is-kubernetes (<a href="https://github.com/kubernetes/website/pull/12065" target="_blank">#12065</a>)</li>
    <li>[ja] translate supported doc versions (<a href="https://github.com/kubernetes/website/pull/12068" target="_blank">#12068</a>)</li>
    <li>[ja] translate setup (<a href="https://github.com/kubernetes/website/pull/12070" target="_blank">#12070</a>)</li>
    <li>[ja] tutorials/index (<a href="https://github.com/kubernetes/website/pull/12071" target="_blank">#12071</a>)</li>
    <li>[ja]translate tutorials/kubernetes-basics (<a href="https://github.com/kubernetes/website/pull/12074" target="_blank">#12074</a>)</li>
    <li>ja-trans: Remove reviewers block from front matter. (<a href="https://github.com/kubernetes/website/pull/12092" target="_blank">#12092</a>)</li>
    <li>[ja] translate cri installation (<a href="https://github.com/kubernetes/website/pull/12095" target="_blank">#12095</a>)</li>
    <li>ja: fix formatting in what is kubernetes (<a href="https://github.com/kubernetes/website/pull/12694" target="_blank">#12694</a>)</li>
    <li>ja-trans: setup/release/building-from-source/ (<a href="https://github.com/kubernetes/website/pull/12721" target="_blank">#12721</a>)</li>
    <li>ja-trans: translate setup/certificates/ (<a href="https://github.com/kubernetes/website/pull/12722" target="_blank">#12722</a>)</li>
    <li>ja-trans: setup/cluster-large/ (<a href="https://github.com/kubernetes/website/pull/12723" target="_blank">#12723</a>)</li>
    <li>ja-trans: setup/minikube/ (<a href="https://github.com/kubernetes/website/pull/12724" target="_blank">#12724</a>)</li>
    <li>ja-trans: setup/multiple-zones/ (<a href="https://github.com/kubernetes/website/pull/12725" target="_blank">#12725</a>)</li>
    <li>ja-trans: setup/node-conformance/ (<a href="https://github.com/kubernetes/website/pull/12728" target="_blank">#12728</a>)</li>
    <li>ja-trans: setup/pick-right-solution/ (<a href="https://github.com/kubernetes/website/pull/12729" target="_blank">#12729</a>)</li>
    <li>ja-trans: setup/scratch/ (<a href="https://github.com/kubernetes/website/pull/12730" target="_blank">#12730</a>)</li>
    <li>ja-trans: setup/custom-cloud/coreos/ (<a href="https://github.com/kubernetes/website/pull/12731" target="_blank">#12731</a>)</li>
    <li>ja-trans: setup/custom-cloud/kops/ (<a href="https://github.com/kubernetes/website/pull/12732" target="_blank">#12732</a>)</li>
    <li>ja-trans: setup/custom-cloud/kubespray/ (<a href="https://github.com/kubernetes/website/pull/12733" target="_blank">#12733</a>)</li>
    <li>ja-trans: setup/independent/control-plane-flags/ (<a href="https://github.com/kubernetes/website/pull/12745" target="_blank">#12745</a>)</li>
    <li>ja-trans: setup/independent/create-cluster-kubeadm/ (<a href="https://github.com/kubernetes/website/pull/12750" target="_blank">#12750</a>)</li>
    <li>ja-trans: setup/independent/high-availability/ (<a href="https://github.com/kubernetes/website/pull/12753" target="_blank">#12753</a>)</li>
    <li>ja-trans: setup/independent/kubelet-integration/ (<a href="https://github.com/kubernetes/website/pull/12754" target="_blank">#12754</a>)</li>
    <li>ja-trans: setup/independent/setup-ha-etcd-with-kubeadm/ (<a href="https://github.com/kubernetes/website/pull/12755" target="_blank">#12755</a>)</li>
    <li>ja-trans: setup/independent/troubleshooting-kubeadm/ (<a href="https://github.com/kubernetes/website/pull/12757" target="_blank">#12757</a>)</li>
    <li>ja-trans: setup/on-premises-metal/krib/ (<a href="https://github.com/kubernetes/website/pull/12770" target="_blank">#12770</a>)</li>
    <li>ja-trans: setup/on-premises-vm/cloudstack/ (<a href="https://github.com/kubernetes/website/pull/12772" target="_blank">#12772</a>)</li>
    <li>ja-trans: setup/on-premises-vm/dcos/ (<a href="https://github.com/kubernetes/website/pull/12780" target="_blank">#12780</a>)</li>
    <li>ja-trans: setup/on-premises-vm/ovirt/ (<a href="https://github.com/kubernetes/website/pull/12781" target="_blank">#12781</a>)</li>
    <li>ja-trans: setup/turnkey/alibaba-cloud/ (<a href="https://github.com/kubernetes/website/pull/12786" target="_blank">#12786</a>)</li>
    <li>ja-trans: setup/turnkey/aws/ (<a href="https://github.com/kubernetes/website/pull/12788" target="_blank">#12788</a>)</li>
    <li>ja-trans: setup/release/notes/ (<a href="https://github.com/kubernetes/website/pull/12791" target="_blank">#12791</a>)</li>
    <li>ja-trans: setup/independent/install-kubeadm.md (<a href="https://github.com/kubernetes/website/pull/12812" target="_blank">#12812</a>)</li>
    <li>ja-trans: setup/turnkey/gce.md (<a href="https://github.com/kubernetes/website/pull/12813" target="_blank">#12813</a>)</li>
    <li>ja-trans: concepts/ (<a href="https://github.com/kubernetes/website/pull/12820" target="_blank">#12820</a>)</li>
    <li>ja-trans: setup/turnkey/clc/ (<a href="https://github.com/kubernetes/website/pull/12824" target="_blank">#12824</a>)</li>
    <li>ja-trans: setup/turnkey/stackpoint/ (<a href="https://github.com/kubernetes/website/pull/12853" target="_blank">#12853</a>)</li>
    <li>ja: format (<a href="https://github.com/kubernetes/website/pull/12866" target="_blank">#12866</a>)</li>
    <li>Translated docs/setup/turnkey/azure.md. (<a href="https://github.com/kubernetes/website/pull/12951" target="_blank">#12951</a>)</li>
  </ul>
</details>
<br>

Co-authored-by: @bells17
Co-authored-by: @hhiroshell
Co-authored-by: @dzeyelid
Co-authored-by: @lkougi
Co-authored-by: @hmaruyama
Co-authored-by: @nasa9084
Co-authored-by: @auifzysr
Co-authored-by: @makocchi-git
Co-authored-by: @inductor

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -153,11 +153,22 @@ contentDir = "content/ko"
 time_format_blog = "2006.01.02"
 language_alternatives = ["en"]
 
+[languages.ja]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName = "日本語 Japanese"
+weight = 4
```
Comment (zacharysarah) [inline config.toml]: Please make sure these weights and file contents are in sync with the current version of `config.toml` in `master`. If accepted right now, this change would create a conflict with other languages.

Comment (cstoku) [reply config.toml]: - Duplicate weights seem to be `Norsk` which is set to a disable language.
  Can I change this Norsk to be the last weight?
- Is it better to change the way to merge `master` into `dev-1.13-ja.1` and edit `config.toml` ?

Comment (zacharysarah) [reply config.toml]: > Can I change this Norsk to be the last weight?

Yes. 👍 

> Is it better to change the way to merge master into dev-1.13-ja.1 and edit config.toml ?

That sounds correct. The goal is for `dev-1.13-ja.1#config.toml` to base on the most current version of `master#config.toml`.

Comment (cstoku) [reply config.toml]: I fixed it.

**Comment by cstoku** [comment]
@zacharysarah Thank you!

**Pull request merged**